conkit/command_line/conkit_plot.py

```diff
@@ -144,6 +144,25 @@ def add_contact_map_chord_args(subparsers):
     subparser.set_defaults(which='contact_map_chord')
 
 
+def add_contact_map_matrix_args(subparsers):
+    description = u"""
+This command will plot a contact map matrix using the provided contacts
+alongside any additional information.
+
+"""
+    subparser = subparsers.add_parser(
+        'cmat',
+        help="Plot a contact map matrix",
+        description=description,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    subparser.add_argument('-e', dest='otherfile', default=None, help='a second contact map to plot for comparison')
+    subparser.add_argument('-ef', dest='otherformat', default=None, help='the format of the second contact map')
```

Comment:
It's an inconsequential detail, but why is the flag for this argument `-e`?

Reply:
I use `-o` already for other bits, so `-e` was just a random choice made ages ago